inventory/group_vars/all/common.yml

```diff
@@ -99,18 +82,14 @@ controller_nodes:
 
 
 ### Messaging
-rabbit_host: 172.17.17.99
-rabbit_hosts: 172.17.17.70:5672,172.17.17.71:5672,172.17.17.72:5672
-rabbit_port: 5672
```

Comment:
I think `rabbit_port` should be in defaults my only concern is when for example we just want a compute node built if the default variable will be added.  I guess we just need to test more.